src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
```

Comment:
1. [VARIABLES] - use variables for the main values so that you'll be able to reuse them and give them descriptive names.